packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -397,7 +397,8 @@ class _CodeViewState extends State<CodeView>
   Widget buildFileSearchField() {
     return wrapInElevatedCard(
       FileSearchField(
-        controller: widget.controller,
+        debuggerController: widget.controller,
+        autoCompleteController: AutoCompleteController(),
```

Comment:
should you create the autoCompleteController once in initState instead of creating it every single build?

Reply:
Switched to no longer passing it in - I think file search widget should own it, not codeview. I've switched back to creating it within the `initState` method for the file search widget. I've also made FileSearchFieldState/autocompleteController public, so they can be accessed from the test.